lib/transport_web/views/error_view.ex

```diff
@@ -1,6 +1,10 @@
 defmodule TransportWeb.ErrorView do
   use TransportWeb, :view
 
+  def render("500.html", %{reason: %Phoenix.Template.UndefinedError{}}) do
+    render(__MODULE__, "internal_server_error.html", %{reason: dgettext("errors", "404: Page not available")})
```

Comment:
This will still return a 500 error right?

Reply:
yep, that was wrong, should be better now!